static/js/place/main_pane.tsx

```diff
@@ -125,7 +125,6 @@ class MainPane extends React.Component<MainPanePropType> {
             locale={this.props.locale}
             highlight={this.props.highlight}
             names={this.props.names}
-            parentPlaces={this.props.parentPlaces}
```

Comment:
The MainPane class still uses parentPlaces in its ChartBlock rendering logic, so I'll leave it there for now. I'm a little hesitant to outright remove it from the Overview, since we're about to revamp these pages and there's a version of the mocks with the parent places still in the Overview box.

Reply:
Update, as discussed offline, also removed ParentPlaces from Overview box.